test/instrumentation/testdata/stable-metrics-list.yaml

```diff
@@ -241,6 +241,24 @@
   - 1310.72
   - 2621.44
   - 5242.88
+- name: statefulset_max_unavailable
+  subsystem: statefulset_controller
+  help: Maximum number of unavailable pods allowed during StatefulSet rolling updates
+  type: Gauge
+  stabilityLevel: BETA
+  labels:
+  - pod_management_policy
+  - statefulset_name
+  - statefulset_namespace
+- name: statefulset_unavailable_replicas
```

Comment:
FYI: we also have this metric

https://github.com/kubernetes/kube-state-metrics/blob/0641627fd5bf6466e16c52dade27d895bfa8985b/internal/store/statefulset.go#L82

Reply:
We may need to point to that in the docs.